### DISCUSSION TITLE
Attachment to new model

### DISCUSSION THREAD

**Discussion by vinod-digitalcubez** [orchidsoftware/platform#2511] [General]
An attachment has already been uploaded and I want to move or copy this attachment with another model table. how can we do this?

**Comment by tabuna** [comment]
In order to attach existing attachments to the model you can use:


```php
use App\Models\User;
use Orchid\Attachment\Models\Attachment;

$user = User::find(1);
$attachment = Attachment::find(1)
 
$user->attachment()->attach($attachment->id);
```

This is the Elocument relationship you can learn about here: https://laravel.com/docs/9.x/eloquent-relationships#updating-many-to-many-relationships
